docs/user/technical-reference/03-10-architecture-updates.md

```diff
@@ -0,0 +1,20 @@
+# Serverless Architecture Updates
+
+## Changes
+
+- The internal Docker Registry is no longer part of the Serverless module. Instead, the Docker Registry is now a separate, standalone module.
```

Comment:
change the bullet points so that it explains what would happen if user switches it on